subprojects/cfa/cfa/src/main/antlr/CfaDsl.g4

```diff
@@ -214,7 +223,7 @@ additiveExpr
 	;
 
 multiplicativeExpr
-	:	ops+=bvConcatExpr (opers+=(MUL | DIV | MOD | REM | BV_MUL | BV_UDIV | BV_SDIV | BV_SMOD | BV_UREM | BV_SREM) ops+=bvConcatExpr)*
+	:	ops+=bvConcatExpr (opers+=(MUL | DIV | MOD | REM | BV_MUL | BV_UDIV | BV_SDIV | BV_SMOD | BV_UREM | BV_SREM | FPREM) ops+=bvConcatExpr)*
```

Comment:
I have one question regarding this. How can you specify the rounding mode?

Reply:
Fixed in [2608512](https://github.com/ftsrg/theta/pull/123/commits/260851260a768790fd16a6f98dcd612f29bb0084)